### PR TITLE
Flattened contracts/ folder for verification on etherscan

### PR DESCRIPTION
**Pull request by maaktweluit**
this is only for verification on etherscan, it does not change the bytecode

### CODE REVIEW

File: contracts/GNTPaymentChannels.sol [hunk 1]
```diff
@@ -1,149 +0,0 @@
-pragma solidity ^0.5.3;
```
Comment (chfast) [inline contracts/GNTPaymentChannels.sol]: Why is this file removed?

Comment (maaktweluit) [reply contracts/GNTPaymentChannels.sol]: it was not used afaik, can put it back if it still has value :)

Comment (chfast) [reply contracts/GNTPaymentChannels.sol]: For me it is ok to remove it, but I would double check someone else.
